File: doc/src/scripts/Basic.java
import java.io.IOException;
import krpc.client.Connection;
import krpc.client.RPCException;
import krpc.client.services.KRPC;

public class Basic {
    public static void main(String[] args) throws IOException, RPCException {
        Connection connection = Connection.newInstance();
        KRPC krpc = KRPC.newInstance(connection);
        System.out.println("Connected to kRPC version " + krpc.getStatus().getVersion());
    }
}

File: doc/src/scripts/Basic.cs
using KRPC.Client;
using KRPC.Client.Services.KRPC;

class Program
{
    public static void Main ()
    {
        using (var connection = new Connection (name : "Example")) {
            var krpc = connection.KRPC ();
            System.Console.WriteLine (krpc.GetStatus ().Version);
        }
    }
}
